configure

```diff
@@ -228,13 +228,15 @@ _have_x86=no
 
 # Add (virtual) features
 add_feature 16bit "16bit color" "_16bit"
+add_feature bink "Bink" "_bink"
```

Comment:
Please mark this line as ResidualVM specific as well so it's not accidentally removed during a merge with ScummVM.